Jenkinsfile

```diff
@@ -52,13 +52,36 @@ try {
           cloverReportDir: '.',
           cloverReportFileName: 'coverage.xml'
         ])
+
+        if (env.CHANGE_ID) { // pull request, then comment it with coding style issues
+          step([
```

Comment:
There is a pipeline step for this (_ViolationsToGitHub_). Please use it instead of generic _step_ or explain why you need it.